src/pages/hotel-card/hotel-card.tsx

```diff
@@ -1,14 +1,21 @@
-import {Hotel} from '../types.ts';
+import { Link } from 'react-router-dom';
+import {Offer, SetCurrentCard} from '../../types.ts';
+import classnames from 'classnames';
 
-export default function HotelCard ({imageSource, price, isBookmarked, cardRating, name, placeCardType}: Hotel): JSX.Element {
-  const buttonActiveClass = isBookmarked ? 'place-card__bookmark-button--active' : '';
-  const bookmarked = isBookmarked ? 'Is bookmarks' : 'To bookmarks';
+export default function HotelCard (offer: Offer, currentCard: boolean, setCurrentCard: SetCurrentCard): JSX.Element {
```

Comment:
Компонент на вход всегда получает объект.

```suggestion
export default function HotelCard (offer: Offer, currentCard: boolean, setCurrentCard: SetCurrentCard): JSX.Element {
```